esi_ui/static/dashboard/project/esi/nodes/esi-nodes-table.controller.js

```diff
@@ -54,6 +75,22 @@
         ctrl.nodesSrc = response.data.nodes;
         ctrl.nodesDisplay = ctrl.nodesSrc;
         console.log(ctrl.nodesSrc);
+
+        var in_provision_transition = false;
+        ctrl.nodesSrc.forEach(function(node) {
+          if (PROVISION_ERROR_STATES.has(node.provision_state)) {
+            return;
+          }
+
+          if (PROVISION_STABLE_STATES.has(node.target_provision_state)) {
+            in_provision_transition = true;
+          }
+        });
+
+        if (in_provision_transition) {
+          $timeout(init, 30000);
```

Comment:
the '30000' should be a constant (and maybe default it to something like ten seconds instead).